### PR TITLE
Console input

### PR DESCRIPTION
**Pull request by EvansSnave**
The console app has been designed to function just like the example in the video.
The app.rb file has been created and serves as the entry point for your console app. It features methods that achieve the following tasks:
Lists all available books.
Lists all registered people (both teachers and students).
Creates a new person (teacher or student, not just a plain Person).
Adds a new book to the catalog.
Establishes a new rental record.
Lists all rentals associated with a specific person ID.
The main.rb file defines the entry point for the application. The main method, which is executed at the end of the file, manages the following processes:
Presents users with a menu of options to choose from.
Allows users to select an option.
Prompts users for any necessary parameters based on their choice.
Provides a way for users to gracefully exit the application.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    puts 'Person created successfully '
+    puts
+    @app.create_a_teacher(name, specialization, age)
+  end
```
Comment (t-yanick) [inline main.rb]: - [ ] Great job working on this milestone. Your main.rb should be just as the entry point into the app as per requiremnents. All your methods should be defined in app.rb. Therefore in your main.rb, you should have something similar to:

```
def main
  app = App.new()

  response = nil

  # *Present the user with a list of options to perform.
  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by entering a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()

```

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parents permission ? [y/n]: '
+    permission = gets.chomp
+    puts 'Person created successfully '
+    puts
+    @app.create_a_student(name, age, permission)
+  end
+
+  def create_a_person
```
Comment (t-yanick) [inline main.rb]: - [ ] Kindly move all the methods and app functionalities to app.rb

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @EvansSnave , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Use of correct gitlow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @EvansSnave , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Use of correct gitlow

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @EvansSnave, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**